### ISSUE TITLE
Venue discussion

### ISSUE THREAD

**Issue by drewby** [ShoCodeJP/ShoCode#2]
I will setup a call/meeting with potential venue [Hayama Marina](https://hayamamarina.com/floorguide/). Capacity is around 60 people and seems price is reasonable for a small event. 

Please list any questions here you think we should ask the venue. I will collect the from the comments and update the list in the issue.

1. What hours we will have access to the venue if we reserve it for the whole day? Does this include setup and cleanup time?
2. What options do we have for coffee/tea service throughout the day?
3. What options do we have for lunch catering? Can we bring in external food such as bento box from vendor?
4. Is there a way to coordinate a BBQ near the venue?
5. What Saturdays are available between August-October?
6. Do you require a deposit to confirm the reservation?

**Comment by chack411** [comment]
Add some confirmation items ...
- Internet connection, bandwidth and charges
- Whether to rent a projector
- The projector specs, resolution, etc.
- Availability of power and power extension cables.

**Comment by chack411** [comment]
In the README, there is a description of `Cost: TBD (goal is under 20,000yen)`

It depends on the cost of the venue, but the cost of 20,000 yen is a little expensive for a one-day event, I guess... If possible, I think that less than 10,000 yen is better.

**Comment by drewby** [comment]
Yes, I agree now that I've looked at a few venue options. My main concern now is food and beverage cost, but I'm sure we can find something reasonable and stay well under 10,000 yen.

We may have the BBQ separate cost?

**Comment by chack411** [comment]
There are some people who do not participate in the BBQ gathering, so I think that the BBQ fee can be charged separately.
